pkg/controller/schedulingmanager/controller.go

```diff
@@ -60,18 +60,18 @@ func StartSchedulerController(config *util.ControllerConfig, stopChan <-chan str
 	restclient.AddUserAgent(kubeConfig, userAgent)
 	client := fedclientset.NewForConfigOrDie(kubeConfig).CoreV1alpha1()
 
-	controller := newController(config, client, stopChan)
+	controller := newController(config, client)
 
 	glog.Infof("Starting scheduler controller")
 	controller.Run(stopChan)
 }
 
-func newController(config *util.ControllerConfig, client corev1alpha1client.CoreV1alpha1Interface, stopChan <-chan struct{}) *SchedulerController {
+func newController(config *util.ControllerConfig, client corev1alpha1client.CoreV1alpha1Interface) *SchedulerController {
 	c := &SchedulerController{
-		scheduler:      make(map[string]schedulingtypes.Scheduler),
 		config:         config,
-		stopChan:       stopChan,
+		scheduler:      make(map[string]schedulingtypes.Scheduler),
```

Comment:
(No action required) Please avoid mixing non-functional changes like ordering with functional changes to simplify both review and future maintenance.

Reply:
Got it. 
The reason here is that old logic is more clear to make `scheduler` near to old `stopChannels`. After I update code based on your comments to delete `stopChannels`, this is not recovered.